mandatory/4-tax.js

```diff
@@ -17,8 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
-
+function addTaxAndFormatCurrency(num) {
+  return "\u00A3" + Number(calculateSalesTax(num)).toFixed(2); 
```

Comment:
`calculateSalesTax` should always return a number so there's no need to use `Number` here either